src/modules/project/pages1/projectFunding/views/fundingSuccess/components/SuccessImageComponent.tsx

```diff
@@ -40,118 +51,186 @@ export const SuccessImageComponent = ({ currentBadge }: { currentBadge?: Badge }
     return null
   }
 
-  const handleCopy = async () => {
-    try {
-      const dataUrl = await getDataUrl()
-      const base64Response = await fetch(dataUrl)
-      const blob = await base64Response.blob()
-      const items = { [blob.type]: blob }
-      const clipboardItem = new ClipboardItem(items)
-      await navigator.clipboard.write([clipboardItem])
-      setCopied(true)
-      setTimeout(() => {
-        setCopied(false)
-        toast.success({
-          title: 'Copied!',
-          description: 'Ready to paste into Social media posts',
-        })
-      }, 1000)
-    } catch {
-      toast.error({
-        title: 'Failed to download image',
-        description: 'Please try again',
-      })
+  const renderSharingStats = () => {
+    if (ambassadorsCount) {
+      return (
+        <>
+          {t('So far, ')}
+          <Body as="span" color="neutral1.12">
+            {ambassadorsCount}
+          </Body>{' '}
+          <Body as="span" regular>
+            {t('ambassador' + (ambassadorsCount === 1 ? ' has' : 's have') + ' enabled')}
+          </Body>{' '}
+          <Body as="span" color="neutral1.12">
+            {totalSats.toLocaleString()}
+          </Body>{' '}
+          {t('sats in contributions to this project.')}
+        </>
+      )
     }
+
+    return ''
   }
 
-  const getDataUrl = async () => {
-    const element = successComponent
-    if (element) {
-      const dataUrl = await htmlToImage.toPng(element, {
-        style: { backgroundColor: 'primary.400', borderStyle: 'double' },
-      })
-      return dataUrl
+  const renderSignInPromptBody = () => {
+    if (!isLoggedIn) {
+      return (
+        <Body
+          size="xl"
+          pt={4}
+          pb={4}
+          textAlign="center"
+          color="neutral1.11"
+          borderTop="1px solid"
+          w="full"
+          borderColor="neutral1.11"
+        >
+          <Link
+            color="primary1.500"
+            textDecoration="underline"
+            onClick={(e) => {
+              e.preventDefault()
+              loginOnOpen()
+            }}
+          >
+            {t('Sign in')}
+          </Link>{' '}
```

Comment:
A link is not required if it is not intended for the purpose of directing to a new URL. Simple text should suffice.